### PR TITLE
pouchdb-errors: remove no-op call to Error constructor

### PR DESCRIPTION
**Pull request by alxndrsn**
Introduced in cef1ded5712dea0256e8fe4d2c9ffaf48c891651.

The original intention is unclear.

### CODE REVIEW

File: packages/node_modules/pouchdb-errors/src/index.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class PouchError extends Error {
   constructor(status, error, reason) {
     super();
```
Comment (SourceR85) [inline packages/node_modules/pouchdb-errors/src/index.js:3]:
```suggestion
    super(reason);
```

**Review by SourceR85 — CHANGES_REQUESTED** [review]
Hi, passing `reason` to parent constructor will initiate `message`.

File: packages/node_modules/pouchdb-errors/src/index.js (hunk 1 above)
Comment (SourceR85) [inline packages/node_modules/pouchdb-errors/src/index.js:3]:
```suggestion
    super(reason);
```

File: packages/node_modules/pouchdb-errors/src/index.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
-    Error(reason);
     this.status = status;
     this.name = error;
     this.message = reason;
```
Comment (SourceR85) [inline packages/node_modules/pouchdb-errors/src/index.js:6]:
```suggestion
```

File: packages/node_modules/pouchdb-errors/src/index.js (hunk 1 above)
Comment (SourceR85) [reply packages/node_modules/pouchdb-errors/src/index.js:3]: Passing reason to parent constructor...

File: packages/node_modules/pouchdb-errors/src/index.js (hunk 2 above)
Comment (SourceR85) [reply packages/node_modules/pouchdb-errors/src/index.js:6]: ... makes message assignment obsolete ;)

**Review by SourceR85 — CHANGES_REQUESTED** [review]
Hi, passing `reason` to parent constructor will initiate `message`.

**Comment by alxndrsn** [comment]
> Hi, passing `reason` to parent constructor will initiate `message`.

Can you help me understand under what circumstances this is true?  I can't see how it would work.  The following looks analogous to me:

```js
class X extends Error {
  constructor() {
    super();
    Error('hello');
  }
};

new X().message
```

which returns empty string...

**Review by SourceR85 — APPROVED** [review]
Ok, I just got more input in Slack chat.
This change will make future changes less noisy (coming soon).

**Comment by SourceR85** [comment]
> Can you help me understand under what circumstances this is true? I can't see how it would work. The following looks analogous to me:
> 
> ```js
> class X extends Error {
>   constructor() {
>     super();
>     Error('hello');
>   }
> };
> 
> new X().message
> ```
> 
> which returns empty string...

Of course,
calling `Error('hello');` will create a new Error instance.
That Error isn't bound to the current `this`-context in constructor, so it's a noop.

The "correct" way to bound "hello" to `this.message` in this context is by calling `super('hello')`.

**Pull request merged**